### DISCUSSION TITLE
What are item duplication fields on the Data Model?

### DISCUSSION THREAD

**Discussion by ched-dev** [directus/directus#12369] [Archive]
I've seen the Item Duplication Fields on the Data Model but haven't found documentation or explanation on what they do. What would I use them for?

<img width="819" alt="Screenshot of Item Duplication Fields on Data Model" src="https://user-images.githubusercontent.com/73146636/160027048-59782c03-bff5-4157-a103-15f40a3a3806.png">

**Comment by azrikahar** [comment]
It is described in the docs here: https://docs.directus.io/configuration/data-model/#duplication 👍 It's helpful when users using "save as copy" wants to specify what exactly is being saved as copy, rather than everything.

Note: You can also try out the search bar on the top right corner of the docs site when searching for keywords. Feel free to let us know if it's not surfacing results properly.
